sql/moz-fx-data-shared-prod/fenix_derived/new_profile_activation_v1/metadata.yaml

```diff
@@ -16,3 +16,16 @@ bigquery:
 scheduling:
   dag_name: bqetl_mobile_activation
   date_partition_parameter: submission_date
+  referenced_tables:
+  - - 'moz-fx-data-shared-prod'
+    - 'org_mozilla_firefox_stable'
+    - 'first_session_v1'
```

Comment:
This is only available after running the SQL generators. Specifying it here explicitly so that if users run generating the DAGs without generating SQL, it is still correctly picked up as dependency